apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx

```diff
@@ -54,29 +63,28 @@ export default class EnrollmentsPanel extends React.Component {
     window.open(`/api/v1/pd/workshops/${workshopId}/enrollments.csv`);
   };
 
-  handleClickMove = () => {
-    this.setState({isMoveEnrollmentsDialogOpen: true});
+  handleClickChangeEnrollments = event => {
+    this.setState({enrollmentChangeDialogOpen: event.target.name});
   };
 
-  handleMoveEnrollmentsCanceled = () => {
+  handleChangeEnrollmentsCanceled = () => {
     this.setState({
-      isMoveEnrollmentsDialogOpen: false
+      enrollmentChangeDialogOpen: null
     });
   };
 
   handleMoveEnrollmentsConfirmed = destinationWorkshopId => {
-    this.setState({
-      isMoveEnrollmentsDialogOpen: false,
-      selectedEnrollments: []
-    });
     this.handleMoveEnrollments(
       destinationWorkshopId,
       this.state.selectedEnrollments
     );
+    this.setState({
```

Comment:
Moved this `setState` down because it appears that shallow test versions of `state` do not follow the "set state happens at the end of a method call" rule.